docs/how-to/http-proxy.md

```diff
@@ -0,0 +1,31 @@
+# How to run Piraeus when using a HTTP proxy (like Squid)
+
+When attempting to deploy Piraeus behind a HTTP proxy, the following configuration needs to be done.
+
+```yaml
+apiVersion: piraeus.io/v1
+kind: LinstorSatelliteConfiguration
+metadata:
+  name: custom-settings # Set a name of the configuration
+spec:
+  podTemplate:
+    spec:
+      containers:
+        - name: linstor-satellite
+          env:
+            - name: HTTP_PROXY
+              value: http(s)://proxy-ip:port
+            - name: HTTPS_PROXY
+              value: http(s)://proxy-ip:port
+            - name: NO_PROXY
+              value: localhost,127.0.0.1,10.0.0.0/8,172.16.0.0/12 # Add internal IP ranges and domains here
+      initContainers:
+        - name: drbd-module-loader
+          env:
+            - name: HTTP_PROXY
+              value: http(s)://proxy-ip:port
+            - name: HTTPS_PROXY
+              value: http(s)://proxy-ip:port
+            - name: NO_PROXY
+              value: localhost,127.0.0.1,10.0.0.0/8,172.16.0.0/12 # Add internal IP ranges and domains here
+```
```

Comment:
Maybe add an explanation why each section is needed? I see the need for the `drbd-module-loader`, because that may call out to `spaas.drbd.io`, but why does the `linstor-satellite` need it?